### DISCUSSION TITLE
Is there an admin access for the open-source version of GitbookIO/gitbook ?

### DISCUSSION THREAD

**Discussion by anjanesh** [GitbookIO/community#726] [1. How to] [answered]
I'm using the new GitbookIO/gitbook locally and not the old GitBook CLI (which was at localhost:4000).
How (where) do I add content for a brand new product documentation ?
I was hoping there to something like localhost:3000/admin or /app etc.

**Comment by addisonschultz** [comment, marked as answer]
Hi @anjanesh - your admin is the same as your published version - as the local version is only serving the frontend of GitBook. 

I.e, you'll need to create and publish a new space on GitBook (app.gitbook.com) and when working on the local frontend, reference the published version of your site. (http://localhost:300/your-published-site.gitbook.io)

**Reply by anjanesh** [reply]
I see - then if I run `npm run build` I can have the entire documentation site locally on my company' private network ?
This product isn't on the public domain and hence we would want the documentation site to also be on our private network instead.

**Reply by addisonschultz** [reply]
Unfortunately the admin side of things will still be served from our side - it's only the front end that would be able to be served from your own private network

**Reply by anjanesh** [reply]
Even if I create the content on app.gitbook.com - can I export the entire documentation on my local and serve it as a website on my private network ?

**Reply by addisonschultz** [reply]
You would be able to export it into a repo - but the content being served to the front end will still need to come from GitBook's servers. Getting it fully local how I believe you're looking to do is possible at this point

**Reply by anjanesh** [reply]
Ah...Okay - I have a bunch of .md files on our network and was hoping to feed that in to GitBook like the way we did for docusaurus in our local network for security reasons since this is not a public product.
